x/auth/vesting/client/cli/tx.go

```diff
@@ -32,102 +25,12 @@ func GetTxCmd() *cobra.Command {
 	}
 
 	txCmd.AddCommand(
-		NewMsgCreateVestingAccountCmd(),
-		NewMsgCreatePermanentLockedAccountCmd(),
 		NewMsgCreatePeriodicVestingAccountCmd(),
 	)
 
 	return txCmd
 }
 
-// NewMsgCreateVestingAccountCmd returns a CLI command handler for creating a
-// MsgCreateVestingAccount transaction.
-func NewMsgCreateVestingAccountCmd() *cobra.Command {
```

Comment:
since it is CLI breaking, can we keep the existing command with deprecated tag?

Reply:
It is minimally breaking (swapping two args), so I'd argue that this isn't necessary.